### PR TITLE
ACRS-33 Choose how to sign-in BRP/UAN

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What? 
[ACRS-33 ](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-33) - Choose how to sign in (BRP/UAN)

## Why? 
Should follow figma design and allow applicant to select different options 

## How? 
- Edited the readme file
- Replace the field label for uan and brp in the fields.json file

## Testing?
tested locally on machine

## Screenshots (optional)
Screenshot 2024-05-03 at 11 20 24

## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: apps/verify/translations/src/en/fields.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
     "legend": "You can use a biometric residence permit (BRP) number or a unique application number (UAN) to sign in to this form.",
     "options": {
       "brp": {
-        "label": "Using a BRP number",
-        "hint": "For example, RZ1234567"
+        "label": "Using a biometric residence permit (BRP) number",
+        "hint": "For example, RZ123456"
```
Comment (jamiecarterHO) [inline apps/verify/translations/src/en/fields.json:7]: I haven't mentioned this to Jake and Luke yet but this BRP example is actually missing a digit and would be invalid - see discussion in the ticket [ACRS-32](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-32)

Comment (TemitopeAyokuHO) [reply apps/verify/translations/src/en/fields.json:7]: and i figured out that the first 2 letter are different from the image as well

File: apps/verify/translations/src/en/fields.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
     "legend": "You can use a biometric residence permit (BRP) number or a unique application number (UAN) to sign in to this form.",
     "options": {
       "brp": {
-        "label": "Using a BRP number",
-        "hint": "For example, RZ1234567"
+        "label": "Using a biometric residence permit (BRP) number",
```
Comment (jamiecarterHO) [inline apps/verify/translations/src/en/fields.json:6]: The text you have changed to is from the older version of the Figma design I think. Could you confirm with UCD and  check we're not updating to an older version?

File: apps/verify/translations/src/en/fields.json [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
+        "hint": "For example, RZ123456"
       },
       "uan": {
-        "label": "Using a UAN",
-        "hint": "For example, 1111-2222-3333-4444"
+        "label": "Using a unique application number (UAN)",
```
Comment (jamiecarterHO) [inline apps/verify/translations/src/en/fields.json:10]: Same here about the older Figma text.

File: README.md [hunk 4]
```diff
@@ -84,4 +70,3 @@ $ yarn run test:lint
 
 ```bash
 $ yarn test:unit
-```
```
Comment (jamiecarterHO) [inline README.md:87]: Need to keep this line to close the bash block

**Comment by jamiecarterHO** [comment]
@TemitopeAyokuHO theres new changes in here that are already in master...can you check with a rebase or merge?

**Comment by jamiecarterHO** [comment]
@TemitopeAyokuHO can we close this as the updates are already merged from elsewhere?

**Comment by TemitopeAyokuHO** [comment]
yes we can close this

**Pull request closed without merging**